docs/articles/documentation/using-testcafe/configuration-file.md

```diff
@@ -610,6 +611,27 @@ If the speed is also specified for an [individual action](../test-api/actions/ac
 *CLI*: [--speed](command-line-interface.md#--speed-factor)  
 *API*: [runner.run({ speed })](programming-interface/runner.md#run)
 
+## clientScripts
+
+Injects scripts from the specified files into each page visited during the tests. Use it to introduce client-side mock functions or helper scripts.
```

Comment:
> Injects scripts from the specified files
> into each page visited during the tests.

The only file names are supported in command line interface. In other interfaces (test api, configuration and programmatic api) both notations (string and object) are available.